wuzz.go

```diff
@@ -1009,25 +1018,27 @@ func (a *App) ParseArgs(g *gocui.Gui, args []string) error {
 			arg_index += 1
 			header := args[arg_index]
 			fmt.Fprintf(vheader, "%v\n", header)
-		case "-d", "--data":
+		case "-d", "--data", "--data-binary":
 			if arg_index == args_len-1 {
-				return errors.New("No POST/PUT value specified")
+				return errors.New("No POST/PUT/PATCH value specified")
 			}
 
-			vmethod, _ := g.View(REQUEST_METHOD_VIEW)
-			setViewTextAndCursor(vmethod, http.MethodPost)
-
 			arg_index += 1
-			add_content_type = true
+			set_data = true
+			set_binary_data = arg == "--data-binary"
 
-			data, _ := url.QueryUnescape(args[arg_index])
+			data := args[arg_index]
+			if !set_binary_data {
+				data, _ = url.QueryUnescape(data)
+			}
```

Comment:
@nwidger  thanks the answer, you are right, `--data-binary` is better, because of compatibility.
(In the future it would be also good to support the `@-` behavior of cURL too)